### PR TITLE
Remove benchmarker CSV input in favor of JSON #946

### PR DESCRIPTION
**Pull request by haddscot**
**Issue #, if available:** #946

**Description of changes:** removed csv input option for benchmarker

**Testing performed:** coverlay

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/benchmark.md [hunk 1]
```diff
@@ -36,25 +36,30 @@ Custom workloads can also be benchmarked with SOCI.
...
 
-We can now run benchmarks on custom workloads using the ```-f``` flag to specify the file path to a csv file containing details of the workloads.
+We can now run benchmarks on custom workloads using the ```-f``` flag to specify the file path to a json file containing details of the workloads.
 
-### Csv file format for custom workloads
+### JSON file format for custom workloads
 
 Ensure that the file being used with the ```-f``` flag follows the following format
 
 ```shell
```
Comment (sondavidb) [inline docs/benchmark.md:56]: minor nit: Can this be ` ```json` instead of ` ```shell`?

**Review by sondavidb — APPROVED** [review]

File: docs/benchmark.md [hunk 2]
```diff
@@ -36,25 +36,30 @@ Custom workloads can also be benchmarked with SOCI.
...
-<Test_name>, <Container_Image_ref>, <Ready_line>, <Soci_Index_Manifest_Digest>
+{
+    "short_name": "<Test_name>"
+    "image_ref": "<Container_Image_ref>"
+    "ready_line": "<Ready_line>"
+    "soci_index_digest": "<Soci_Index_Manifest_Digest>"
+}
 ```
 
 Example :
```
Comment (austinvazquez) [inline docs/benchmark.md:65]: Lets update the example for the JSON format.

File: docs/benchmark.md (hunk 1 above)
Comment (haddscot) [reply docs/benchmark.md:56]: 👍

File: benchmark/utils.go [hunk 3]
```diff
@@ -113,11 +111,7 @@ func GetImageList(file string) ([]ImageDescriptor, error) {
 	if jsonErr == nil {
 		return images, nil
 	}
-	images, csvErr := GetImageListFromCsv(f)
-	if csvErr == nil {
-		return images, nil
-	}
-	return nil, errors.Join(jsonErr, csvErr)
+	return nil, jsonErr
```
[the remark this answers is not in the document]
Comment (haddscot) [reply benchmark/utils.go:114]: 👍

File: docs/benchmark.md (hunk 2 above)
Comment (haddscot) [reply docs/benchmark.md:65]: 👍

**Review by austinvazquez — APPROVED** [review]

**Review by sondavidb — APPROVED** [review]

**Pull request merged**